src/pages/vereniging/besturen.jsx

```diff
@@ -24,6 +24,7 @@ const Bestuur = (props) => {
 
 
 const BoardsList = styled.div`
+  margin-top: 10pt;
```

Comment:
What is this 'pt' thingy? Maybe we can better use 'em' for units, as this is the most used relative unit accroding to [MDN](https://developer.mozilla.org/en-US/docs/Learn/CSS/Introduction_to_CSS/Values_and_units)

Reply:
A thing @Agarwaemben pointed me to :)